packages/web-app-text-editor/src/App.vue

```diff
@@ -91,8 +91,9 @@ export default {
       unref(defaults)
         .getFileInfo(filePath, [DavProperty.Permissions])
         .then((response) => {
-          isReadOnly.value =
-            response.fileInfo[DavProperty.Permissions].indexOf(DavPermission.Updateable) === -1
+          isReadOnly.value = ![DavPermission.Updateable, DavPermission.FileUpdateable].includes(
```

Comment:
`includes` doesn't work that way 😅 

```js
![DavPermission.Updateable, DavPermission.FileUpdateable].some(p => response.fileInfo[DavProperty.Permissions].indexOf(p) > -1)
```
Need to work with a callback based solution here.

Reply:
Why? Isn’t it a exact match?

Reply:
nope. the permission string in the file info is a concatenation of multiple permissions, e.g. `NVM`, and we want to check if either `NV`("Updatable") or `W` ("FileUpdatable`) is part of that permission string.

Reply:
thx, fixed